File: MetricsCalculators/test/cpp/Product.cpp
#include "Product.h"

Product::Product(string name, double cost){
	this->name = name;
	this->cost = cost;

}

string Product::getName() const {
	return name;
}

double Product::getCost() const {
	return cost;
}

bool Product::compareTo(const Product &prod){
	return this->name < prod.name;
}

File: MetricsCalculators/test/java/src/main/java/store/Product.java
package store;

public class Product {
	String name;
	double cost;

	public Product(String name, double cost) {
		this.name = name;
		this.cost = cost;
	}

	public String getName() {
		return name;
	}

	public double getCost() {
		return cost;
	}

	public boolean compareTo(Product product) {
		return name.compareTo(product.name) < 0;
	}

}
